app/views/workflows/show.json.jbuilder

```diff
@@ -10,7 +10,9 @@ json.folder_contents (@workflow.folder_contents) do |content|
   json.set! 'type', Pathname(content).file? ? 'file' : 'dir'
   json.set! 'fsurl', Filesystem.new.fs(content)
   json.set! 'fs_base', Filesystem.new.fs(File.dirname(content))
-  json.set! 'terminal_base', OodAppkit.shell.url(path: File.dirname(content)).to_s
+  json.set! 'terminal_base', OODClusters[@workflow.batch_host] && OODClusters[@workflow.batch_host].login_allow? ?
+      OodAppkit.shell.url(path: File.dirname(content), host: @workflow.batch_host).to_s :
```

Comment:
Is this correct?

```ruby
host: @workflow.batch_host
```

instead of...

```ruby
host: OODClusters[@workflow.batch_host].login.host
```

I think since we have aliases set up for `oakley`, `ruby`, and `owens`, that it doesn't affect us. But we should use the specified login host in the cluster config just in case this isn't true for another HPC center.